index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=1024px, initial-scale=1.0">
+  <title>Valentina Krivelskaya</title>
```

Comment:
Optional: from my point of view, something like Valentina Krivelskaya CV will be more logical :)